pkg/assert/assert.go

```diff
@@ -10,3 +10,10 @@ func NoError(err error, description string) {
 		testsuit.T.Errorf("%s, \n err: %v", description, err)
 	}
 }
+
```

Comment:
Haven't we agreed on removing this package?

Reply:
And Why so ?  I see no harm :thinking: